pages/sitemap.xml.njk

```diff
@@ -1,14 +1,14 @@
 ---
 permalink: /sitemap.xml
 eleventyExcludeFromCollections: true
+layout: layouts/pure
```

Comment:
The `pure` layout just renders the content without putting anything extra around it. This is definitely the one we want for a sitemap.